packages/cli-plugin-metro/launchPackager.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
Do we still need those scripts?
I don't see any benefit in having those files separated.
Can we try to mimic those inside execa calls?
I see that at least `cwd` can be set without using it from `PROJECT_ROOT`

Reply:
Due to `open` limitations we need this additional step.
Feel free to resolve this.